doc/source/whatsnew/v0.18.2.txt

```diff
@@ -74,3 +74,7 @@ Performance Improvements
 
 Bug Fixes
 ~~~~~~~~~
+
+- Bug in ``PeriodIndex.resample`` not changing ``freq`` when empty
+(:issue:`13067`)
+- Bug in ``PeriodIndex._shallow_copy`` returning a ``float64`` index
```

Comment:
say ``PeriodIndex`` construction. remember a user will be reading these and has no clue about internals